kube-client/Cargo.toml

```diff
@@ -42,6 +42,7 @@ dirs = { package = "dirs-next", optional = true, version = "2.0.0" }
 serde = { version = "1.0.130", features = ["derive"] }
 serde_json = "1.0.68"
 serde_yaml = { version = "0.8.21", optional = true }
+serde_with = { version = "2.0.1" }
```

Comment:
i think this is fine to add.. it's a [fairly popular crate](https://crates.io/crates/serde_with), but you might be able to turn off a lot of [its features](https://github.com/jonasbb/serde_with/blob/master/serde_with/Cargo.toml#L26-L59)?